admin_app/src/app/dashboard/components/Overview.tsx

```diff
@@ -1,7 +1,95 @@
 import React from "react";
 import { Box } from "@mui/material";
+import { StatCard, StatCardProps } from "@/app/dashboard/components/StatCard";
+import ForumIcon from "@mui/icons-material/Forum";
+import SupportAgentIcon from "@mui/icons-material/SupportAgent";
+import NewReleasesOutlinedIcon from "@mui/icons-material/NewReleasesOutlined";
+import ThumbUpIcon from "@mui/icons-material/ThumbUp";
+import ThumbDownIcon from "@mui/icons-material/ThumbDown";
+import { Period } from "../types";
+import { useAuth } from "@/utils/auth";
+import { getStatsCardData } from "@/app/dashboard/api";
+import { useEffect } from "react";
+
+interface OverviewProps {
+  timePeriod: Period;
+}
+
+const Overview: React.FC<OverviewProps> = ({ timePeriod }) => {
+  const { token } = useAuth();
+  const [statCardData, setStatCardData] = React.useState<StatCardProps[]>([]);
+  useEffect(() => {
+    getStatsCardData(timePeriod, token!).then((data) => {
+      const {
+        content_feedback_stats,
+        query_stats,
+        response_feedback_stats,
+        urgency_stats,
+      } = data.stats_cards;
+
+      console.log(data.stats_cards);
+
+      parseCardData(data, timePeriod);
+      // Total Queries
```

Comment:
Seems like this comment should be deleted as well

Reply:
Thanks! will do